components/Cards/index.js

```diff
@@ -17,3 +17,90 @@
 // </div>
 //
 // Create a card for each of the articles and add the card to the DOM.
+
+// *** Axios Get *** //
+
+axios.get('https://lambda-times-backend.herokuapp.com/articles')
+.then( data => {
+console.log('It is Working!', data);
+
+const articles = data.data.articles;
+const Cards = document.querySelector('.cards-container');
+console.log(articles);
+
+// *** Bootstrap *** //
+
+articles.bootstrap.forEach(card => {
+    Cards.appendChild(createCard(card.headline, card.authorPhoto, card.authorName));
+    })
+
+// *** Javascript *** //
+
+articles.javascript.forEach(card => {
+    Cards.appendChild(createCard(card.headline, card.authorPhoto, card.authorName));
+    })
+
+// *** Jquery *** //
+
+articles.jquery.forEach(card => {
+    Cards.appendChild(createCard(card.headline, card.authorPhoto, card.authorName));
+    })
+
+// *** Node *** //
+
+articles.node.forEach(card => {
+    Cards.appendChild(createCard(card.headline, card.authorPhoto, card.authorName));
+    })
+
+// *** Tech *** //
+
+articles.technology.forEach(card => {
+    Cards.appendChild(createCard(card.headline, card.authorPhoto, card.authorName));
+    })
+    })
+
+// *** Neg *** //
+
+.catch( error => {
+    console.log("This ain't working bro", error);
+    })
+
```

Comment:
YES! Great job on the axios Get request to retrieve data, followed by the looping of individual arrays to create articles. Perfect. If you  want to make your code dry you can loop through an object using a `for...in loop`. like this 
```
axios
  .get('https://lambda-times-backend.herokuapp.com/articles')
  .then(response => {
    let articles = response.data.articles;
    for (topic in articles) {
      articles[topic].forEach(article => {
        entry.appendChild(ArticleCard(article));
      });
    }
  });
```